ops/stand-alone/install-ekl.sh

```diff
@@ -14,11 +14,11 @@ echo "deb https://artifacts.elastic.co/packages/6.x/apt stable main" |  tee -a /
 apt-get update
 
 # Elastic Search
-apt-get install -y elasticsearch
+apt-get install -y elasticsearch=6.1.3
 service elasticsearch start
 
 # Kibana
-apt-get install -y kibana
+apt-get install -y kibana=6.1.3
 cat << EOF > /etc/kibana/kibana.yml
 server.host: "0.0.0.0"
```

Comment:
127.0.0.1

Reply:
0.0.0.0 is needed to let kibana listen other hosts that are not `localhost`.

Reply:
that is exactly why it is wrong

Reply:
you always have to listen on the specified ip/network card you want to expose. by default put 127.0.0.1 since this is for local testing. if you want to expose kibana to the outside world, that should be done by ansible/chef/puppet or the shellscript that deploys it.